benchmark/src/bin/lisar.rs

```diff
@@ -0,0 +1,74 @@
+use benchmark::{
+    bin_utils::{
+        create_benchmarker_from_opt, linear_search_max_throughput, try_start_metrics_server,
+    },
+    ruben_opt::{RubenOpt, TransactionPattern},
+    txn_generator::{LoadGenerator, PairwiseTransferTxnGenerator, RingTransferTxnGenerator},
+};
+use logger::{self, prelude::*};
+use std::ops::DerefMut;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+#[structopt(
+    name = "LiSar",
+    author = "Libra",
+    about = "LiSar (Li)near (S)e(ar)ch Maximum Throughput."
+)]
+pub struct LisarOpt {
+    #[structopt(flatten)]
+    pub ruben_opt: RubenOpt,
+    /// Upper bound value of submission rate for each client.
+    #[structopt(short = "u", long = "upper_bound", default_value = "100")]
```

Comment:
If the goal of this binary is to find the max throughput, what's the purpose of having an upper bound? Or should we at least have the default value of upper_bound to be a very large number?